src/components/studies/participants/ParticipantDeleteModal.tsx

```diff
@@ -73,33 +74,37 @@ const ParticipantDeleteModal: FunctionComponent<ParticipantDeleteModalProps> = (
     } = useStudy(studyId)
 
     const {data} = useParticipants(study?.identifier, currentPage, pageSize, tab,)
-    const {mutateAsync} = useUpdateParticipantInList()
+    const {mutateAsync, error: deleteParticipantError, isSuccess} = useUpdateParticipantInList()
     const invalidateParticipants = useInvalidateParticipants()
-      
+    const [error, setError] = React.useState<Error>()
+    React.useEffect(()=>{
```

Comment:
do we need `error` state variable? Can we just use deleteParticipantError?

Reply:
I had it set up this way because when trying to get the message in JSX, deleteParticipantError was acknowledged as an unknown object and to remove the banner when the modal was closed. Otherwise the alert would still be there when opening the modal.

Reply:
got it